### ISSUE TITLE
[FileFormats.CBF] bug copying unsupported variable bounds to CBF

### ISSUE THREAD

**Issue by odow** [jump-dev/MathOptInterface.jl#2570]
Reported as https://discourse.julialang.org/t/jump-moi-wrapper-interface-problem/121776/5

```Julia
julia> import MathOptInterface as MOI

julia> model = MOI.Utilities.Model{Float64}()
MOIU.Model{Float64}
├ ObjectiveSense: FEASIBILITY_SENSE
├ ObjectiveFunctionType: MOI.ScalarAffineFunction{Float64}
├ NumberOfVariables: 0
└ NumberOfConstraints: 0

julia> x, _ = MOI.add_constrained_variable(model, MOI.GreaterThan(1.0))
(MOI.VariableIndex(1), MathOptInterface.ConstraintIndex{MathOptInterface.VariableIndex, MathOptInterface.GreaterThan{Float64}}(1))

julia> dest = MOI.FileFormats.CBF.Model()
MOI.FileFormats.CBF.Model
├ ObjectiveSense: FEASIBILITY_SENSE
├ ObjectiveFunctionType: MOI.ScalarAffineFunction{Float64}
├ NumberOfVariables: 0
└ NumberOfConstraints: 0

julia> MOI.copy_to(dest, model)
MathOptInterface.Utilities.IndexMap with 2 entries:
  MOI.VariableIndex(1)                                    => MOI.VariableIndex(1)
  ConstraintIndex{VariableIndex, GreaterThan{Float64}}(1) => ConstraintIndex{VariableIndex, GreaterThan{Float64}}(1)

julia> print(dest)
Feasibility

Subject to:

VariableIndex-in-GreaterThan{Float64}

```

**Comment by blegat** [comment]
I'm not sure to understand why we need this. If it's not supported, `add_constraint` is already throwing `UnsupportedConstraint` so we shouldn't have to call `supports_constraint`. If the model does not throw correctly then the model should be fixed, we should not complicate `copy_to` just to make bugs occur less frequently (as they will still appear if the user calls `add_constraint` directly).

**Comment by odow** [comment]
Okay, I guess we could fix this in `GenericModel` then.

**Comment by odow** [comment]
Closed by #2572
